actionview/lib/action_view/helpers/form_tag_helper.rb

```diff
@@ -466,7 +485,12 @@ def check_box_tag(name, value = "1", checked = false, options = {})
       #
       #   radio_button_tag 'color', "green", true, class: "color_input"
       #   # => <input checked="checked" class="color_input" id="color_green" name="color" type="radio" value="green" />
-      def radio_button_tag(name, value, checked = false, options = {})
+      def radio_button_tag(name, value, *args)
+        if args.length >= 3
+          raise ArgumentError, "wrong number of arguments (given #{args.length + 1}, expected 1..3)"
```

Comment:
`radio_button_tag` has 2 mandatory args (`name` and `value`) and up to 2 optional args (`checked` and `options`), so this should be:

```suggestion
          raise ArgumentError, "wrong number of arguments (given #{args.length + 2}, expected 2..4)"
```

(Incidentally, the reason I wrote `if args.length + 1 > 4` in https://github.com/rails/rails/pull/45527#discussion_r922277156 was for consistency with `given #{args.length + 1}` and `expected 1..4`.  In other words, here, `if args.length + 2 > 4` would be consistent with `given #{args.length + 2}` and `expected 2..4`.)